### DISCUSSION TITLE
Re-crear rama para hacer tests

### DISCUSSION THREAD

**Discussion by CarmenRuizPorcel** [Aparking/AparKing_Backend#77] [General]
Para seguir lo establecido en el Configuration Management, se ha vuelto a crear la rama de createModels para realizar los tests ahí ya que se había borrado al terminar de implementar la funcionalidad. 
En el caso de que le pase el mismo problema a alguien, esta es la solución que me han dado desde coordinación.

**Comment by laurolmer** [comment]
Perfecto Carmen, lo tendremos en cuenta gracias
